### ISSUE TITLE
What is a reasonable loss for BLIP-2 second stage training?

### ISSUE THREAD

**Issue by linzhiqiu** [salesforce/LAVIS#552]
I tried to finetune BLIP2 using the official lavis/projects/blip2/train/pretrain_stage2.yaml on COCO+VG+CC3M+CC12M+SBU, but my loss stucks at 3.5-3.7 after 80K iterations. Is this expected?
